fyle_intacct_api/utils.py

```
@@ -14,6 +14,18 @@ def assert_valid(condition: bool, message: str) -> Response or None:
             'message': message
         })
 
+
+def invalidate_intacct_credentials(workspace_id, sage_intacct_credentials=None):
+    if not sage_intacct_credentials:
+        sage_intacct_credentials = SageIntacctCredential.objects.filter(workspace_id=workspace_id, is_expired=False).first()
```

Ruff (0.8.2)

Undefined name `SageIntacctCredential`

(F821)